sys/net/link_layer/ieee802154/ieee802154_frame.c

```diff
@@ -252,14 +208,14 @@ uint8_t ieee802154_frame_read(uint8_t *buf, ieee802154_frame_t *frame,
             frame->src_addr[0] = buf[index + 7];
             index += 8;
             break;
-        }
```

Comment:
Mh… There might be a compiler warning with `-Wall` nevertheless… or was this only with enums?